src/Catalyst.Core.Modules.Dfs.Tests/UnitTests/ExceptionAssert.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Catalyst.Core.Modules.Dfs.Tests.UnitTests
+{
+    /// <summary>
+    ///   Asserting an <see cref="Exception"/>.
+    /// </summary>
+    public static class ExceptionAssert
+    {
+        public static T Throws<T>(Action action, string expectedMessage = null) where T : Exception
+        {
+            try
+            {
+                action();
+            }
+            catch (AggregateException e)
+            {
+                var match = e.InnerExceptions.OfType<T>().FirstOrDefault();
+                if (match != null)
+                {
+                    if (expectedMessage != null)
+                        Assert.AreEqual(expectedMessage, match.Message, "Wrong exception message.");
+                    return match;
+                }
+
+                throw;
+            }
+            catch (T e)
+            {
+                if (expectedMessage != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)